tests/pos/i5720.scala

```diff
@@ -0,0 +1,5 @@
+class A[B[_] <: Long]
```

Comment:
Can you confirm that we get a failed test for this file without the fix ?

Reply:
Yes.